### DISCUSSION TITLE
Github pages not connecting with custom domain

### DISCUSSION THREAD

**Discussion by wyanido** [community/community#23479] [Other Feature Feedback, Questions, & Ideas] [answered]
<p>I’m trying to configure Github Pages using a custom domain for the first time, but despite the guides I follow, I haven't had any success in the site content displaying. On the config tab it shows success in publishing but the url itself doesn’t display anything on any browser or even the 404 page.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/6/462ee0f3606517576c56463dc0597c1761e7989a.png" data-download-href="/uploads/short-url/a0RRkgOx17nY11SZ5SOqvHrJX06.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/80712417/181077327-bcfad241-7aa5-4334-8a27-198f1b939748.png" alt="image" data-base62-sha1="a0RRkgOx17nY11SZ5SOqvHrJX06" width="690" height="65" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/6/462ee0f3606517576c56463dc0597c1761e7989a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">775×74 1.65 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>It <em>does</em> say the domain is improperly configured (InvalidDNSError) along with a suggestion on what CNAME record to add, but this record is already in place along with other A records confirmed to be working through $ dig.<br>
<img src="https://user-images.githubusercontent.com/80712417/181077328-5a39c9c2-015b-41df-aaed-96714d4ca0a3.png" alt="image" data-base62-sha1="gTXfvFfV7ilC4UgQcSwSjfvuwiJ" width="513" height="71">
<p>Any advice is appreciated. I can provide more info where necessary.</p>

**Comment by Akash1134** [comment]
<p>Welcome to the GitHub community, <a class="mention" href="/nidoshark">@nidoshark</a> We’re happy you’re here. You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails–giving a few more details might help someone give you a nudge in the right direction. I’ve gone ahead and moved it for you. Good luck!</p>

**Comment by yoannchaudet** [comment, marked as answer]
<p>Hello <a class="mention" href="/nidoshark">@nidoshark</a> and welcome here,</p>
<p>You created <code>A</code> records for <code>mnstrshark.com</code> but configured your custom domain as <code>www.mnstrshark.com</code>.</p>
<p>You are therefore missing a record for <code>www</code> in your DNS zone. You can fix that by creating a <code>CNAME</code> record for <code>www</code> pointing to <code>nidoshark.github.io</code> as documented <a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain-and-the-www-subdomain-variant">here</a>.</p>
<p>When this is done and after sometime for the DNS change to propagate, visit again the settings page for your Pages site. This will trigger the DNS detection change and issue you a TLS certificate.</p>
<p>Hope that helps,</p>
